src/js/tabs/vtx.js

```diff
@@ -960,8 +960,9 @@ TABS.vtx.initialize = function (callback) {
             powersString += "[" + power_list[index].value + "]=" + power_list[index].label + ", ";
         }
         powersString += "}\n";
-        let text = frequencieString + freqBandsString + bandsString + powersString;
-        return text;
+        const deviceTable = "deviceTable = { [1]=\"6705\", [3]=\"SA\", [4]=\"Tramp\", [255]=\"None\" },\n";
+        const pitModeTable = "pitModeTable = { [0]=\"OFF\", \"ON\" },\n";
+        return `return {\n ${frequencieString} ${freqBandsString} ${bandsString} ${powersString} ${deviceTable} ${pitModeTable} }`;
```

Comment:
You are adding now an space between each value, that was not added when you used the `+ ` concatenation. Is that the desired result?

Reply:
No, no it wasn't, good spot...